src/parser/scanner_support.c

```diff
@@ -510,6 +510,23 @@ pt_get_hint (const char *text, PT_HINT hint_table[], PT_NODE * node)
 		  node->info.update.hint = (PT_HINT_ENUM) (node->info.update.hint | hint_table[i].hint);
 		}
 	      break;
+	    case PT_HINT_NO_SUPPLEMENTAL_LOG:	/* no supplemental_log */
+	      if (node->node_type == PT_UPDATE)
+		{
+		  node->info.update.hint = (PT_HINT_ENUM) (node->info.update.hint | hint_table[i].hint);
+		}
+	      else if (node->node_type == PT_DELETE)
+		{
+		  node->info.delete_.hint = (PT_HINT_ENUM) (node->info.delete_.hint | hint_table[i].hint);
+		}
+	      else if (node->node_type == PT_INSERT)
+		{
+		  node->info.insert.hint = (PT_HINT_ENUM) (node->info.insert.hint | hint_table[i].hint);
+		}
+
+	      hint_table[i].arg_list = NULL;
```

Comment:
Could you tell me why this is necessary? some hints initialize it and some don't.

Reply:
I just follow the NO_LOGGING hint, and I found that this function(pt_get_hint) are not called anywhere. So I will discard this change.

Reply:
I doubt it's good to ignore `pt_get_hint()`. Although it is not used now, this seems a utility function to be able to be used later. Plus, it is not surrounded by macro `ENABLE_UNUSED_FUNCTION`. I'd say it should be considered if it is not that picky to handle.